contract-service/src/main/java/com/example/contractservice/contract/entity/CommissionsCapacity.java

```diff
@@ -0,0 +1,37 @@
+package com.example.contractservice.contract.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@Table(name = "commissions_capacity")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class CommissionsCapacity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "commission_code", nullable = false, unique = true, columnDefinition = "CHAR(36)")
+    private String commissionCode;
+
+    @Column(name = "recruit_capacity", nullable = false)
+    private int recruitCapacity;
+
+    @Column(name = "recruited_count", nullable = false)
+    private int recruitedCount;
```

Comment:
`recruit`라는 표현이 `뽑는다`라는 의미로 해석될 수 있기 때문에
지원 단계와 선정 단계 중 어떤 의미를 가지는지 혼동될 여지가 있을 것 같아요. 
혹시 `apply`라는 표현은 어떨까요?

```suggestion
    @Column(name = "apply_capacity", nullable = false)
    private int applyCapacity; // 지원 정원

    @Column(name = "applied_count", nullable = false)
    private int appliedCount; // 현재까지 지원한 인원
```

Reply:
해당 이름으로 ERD 변경 완료하였고, 코드 수정 하겠습니다!